### PR TITLE
feat: rewrite docs and physx raycast playground

### PR DESCRIPTION
**Pull request by yangfengzzz**

### CODE REVIEW

File: docs/collider.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+} from 'oasis-engine';
+import { PhysXPhysics } from "@oasis-engine/physics-physx";
+
+const engine = new WebGLEngine("canvas", PhysXPhysics);
+
+// create sphere test entity
+let sphereEntity = rootEntity.createChild('SphereEntity');
+sphereEntity.position = new Vector3(-2, 0, 0);
+
+let sphereCollider = sphereEntity.addComponent(StaitcCollider);
```
Comment (cptbtptpbcptdtptp) [inline docs/collider.md:63]: StaitcCollider

File: docs/collider.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+backends when you initialize the engine according to the physical features you use. The lite package is more
+lightweight, but supports fewer physical features. The physX package is powerful, but packaged The latter mention is
+larger, and the code is as follows:
+
+```typescript
+import {
+  Vector3,
+  SphereColliderShape,
+  BoxColliderShape,
+  DynamicCollider,
```
Comment (cptbtptpbcptdtptp) [inline docs/collider.md:52]: StaitcCollider

File: docs/collider.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import { PhysXPhysics } from "@oasis-engine/physics-physx";
+
+const engine = new WebGLEngine("canvas", PhysXPhysics);
+
+// create sphere test entity
+let sphereEntity = rootEntity.createChild('SphereEntity');
+sphereEntity.position = new Vector3(-2, 0, 0);
+
+let sphereCollider = sphereEntity.addComponent(StaitcCollider);
+let shphereColliderShape = new SphereColliderShape()
```
Comment (cptbtptpbcptdtptp) [inline docs/collider.md:64]: sphereColliderShape

File: docs/collider.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+shphereColliderShape.radius = 1.25;
+sphereCollider.addColliderShape(shphereColliderShape);
+
+// create box test entity
+let boxEntity = rootEntity.createChild('BoxEntity');
+let boxCollider = boxEntity.addComponent(DynamicCollider);
+let boxColliderShape = new BoxColliderShape()
+boxColliderShape.setSize(2.0, 2.0, 2.0);
+boxCollider.addColliderShape(boxColliderShape);
```
Comment (cptbtptpbcptdtptp) [inline docs/collider.md:73]: addShape

File: docs/collider.md (hunk 1 above)
Comment (yangfengzzz) [reply docs/collider.md]: done

File: docs/collider.md (hunk 2 above)
Comment (yangfengzzz) [reply docs/collider.md:53]: done

File: docs/collider.md (hunk 3 above)
Comment (yangfengzzz) [reply docs/collider.md]: done

File: docs/collider.md (hunk 4 above)
Comment (yangfengzzz) [reply docs/collider.md]: done

**Review by eyworldwide — APPROVED** [review]
+1

**Pull request merged**